### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by radhikaj**

### CODE REVIEW

**Review by CyanDevs — APPROVED** [review]

**Comment by ghost** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/openenclave/openenclave-ci/pull/32#pullrequestreview-688841760" title="Approved">CyanDevs</a>*, *<a href="https://github.com/openenclave/openenclave-ci/pull/32#" title="Author self-approved">radhikaj</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **brmclaren**
You can assign the PR to them by writing `/assign @brmclaren` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=openenclave%2Fopenenclave-ci).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/openenclave/openenclave-ci/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
